containers/ddev-webserver/tests/ddev-webserver/php_webserver.bats

```diff
@@ -85,7 +81,7 @@
     ;;
   8.4)
     # TODO: Update this list as more become available
-    extensions="bcmath bz2 curl gd intl ldap mbstring mysqli pgsql readline soap sqlite3 xml zip"
+    extensions="bcmath bz2 curl gd intl json ldap mbstring mysqli pgsql readline soap sqlite3 xml zip"
```

Comment:
I reverted added extensions here, because they are not loaded in `ddev-webserver`.
(Need to understand how to use `phpenmod` with custom PHP version.